cloud/shared/bin/lib/health.sh

```diff
@@ -6,7 +6,7 @@
 #   1: The url for the healthcheck endpoint
 #   2: (Optional) Amount of time to wait before timing out, default
 #######################################
-function health::wait_for_success() {
+function health::wait_for_healthy() {
```

Comment:
**todo:** Based on the change you made to use playIndex in cloud/azure/bin/deploy you don't need the changes in this file anymore?